2018-07-10 10:36:43.143155: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2
2018-07-10 10:36:44.324912: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:898] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-07-10 10:36:44.325814: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1356] Found device 0 with properties: 
name: GeForce GTX TITAN X major: 5 minor: 2 memoryClockRate(GHz): 1.076
pciBusID: 0000:05:00.0
totalMemory: 11.93GiB freeMemory: 11.81GiB
2018-07-10 10:36:44.500564: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:898] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-07-10 10:36:44.501308: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1356] Found device 1 with properties: 
name: GeForce GTX 980 Ti major: 5 minor: 2 memoryClockRate(GHz): 1.1395
pciBusID: 0000:07:00.0
totalMemory: 5.93GiB freeMemory: 5.83GiB
2018-07-10 10:36:44.501622: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1435] Adding visible gpu devices: 0, 1
2018-07-10 10:36:45.076694: I tensorflow/core/common_runtime/gpu/gpu_device.cc:923] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-07-10 10:36:45.076750: I tensorflow/core/common_runtime/gpu/gpu_device.cc:929]      0 1 
2018-07-10 10:36:45.076768: I tensorflow/core/common_runtime/gpu/gpu_device.cc:942] 0:   N Y 
2018-07-10 10:36:45.076779: I tensorflow/core/common_runtime/gpu/gpu_device.cc:942] 1:   Y N 
2018-07-10 10:36:45.077215: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1053] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 11436 MB memory) -> physical GPU (device: 0, name: GeForce GTX TITAN X, pci bus id: 0000:05:00.0, compute capability: 5.2)
2018-07-10 10:36:45.245596: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1053] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 5609 MB memory) -> physical GPU (device: 1, name: GeForce GTX 980 Ti, pci bus id: 0000:07:00.0, compute capability: 5.2)
Using TensorFlow backend.
Uppercase local vars:
	BATCH_SIZE: 30
	CRITIC_ITERS: 5
	DIM: 64
	ITERS: 300000
	LAMBDA: 10
	MODE: wgan-gp
	OUTPUT_DIM: 230400
Traceback (most recent call last):
  File "../../gan_ucf101.py", line 185, in <module>
    train_gen = lib.ucf101.load_keras_gen(BATCH_SIZE)
  File "/home/linkermann/opticalFlow/opticalFlowGAN/tflib/ucf101.py", line 119, in load_keras_gen
    train_generator, validation_generator = get_generators()
  File "/home/linkermann/opticalFlow/opticalFlowGAN/tflib/ucf101.py", line 34, in get_generators
    class_mode=None)
  File "/home/linkermann/miniconda2/envs/MA/lib/python3.5/site-packages/keras_preprocessing/image.py", line 935, in flow_from_directory
    interpolation=interpolation)
  File "/home/linkermann/miniconda2/envs/MA/lib/python3.5/site-packages/keras_preprocessing/image.py", line 1681, in __init__
    for subdir in sorted(os.listdir(directory)):
FileNotFoundError: [Errno 2] No such file or directory: '../data/train/'
